phpBB/phpbb/console/command/searchindex/list_all.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\console\command\searchindex;
+
+use phpbb\config\config;
+use phpbb\console\command\command;
+use phpbb\di\service_collection;
+use phpbb\language\language;
+use phpbb\user;
+use Symfony\Component\Console\Command\Command as symfony_command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class list_all extends command
+{
+	/** @var config */
+	protected $config;
+
+	/** @var language */
+	protected $language;
+
+	/** @var service_collection */
+	protected $search_backend_collection;
+
+	/**
+	 * Construct method
+	 *
+	 * @param config					$config
+	 * @param language					$language
+	 * @param service_collection		$search_backend_collection
+	 * @param user						$user
+	 */
+	public function __construct(config $config, language $language, service_collection $search_backend_collection, user $user)
+	{
+		$this->config = $config;
+		$this->language = $language;
+		$this->search_backend_collection = $search_backend_collection;
+
+		parent::__construct($user);
+	}
+
+	/**
+	 * Sets the command name and description
+	 *
+	 * @return void
+	 */
+	protected function configure()
+	{
+		$this
```

Comment:
Change this section to look as follows:
```
		$this->setName('searchindex:list')
			->setDescription($this->language->lang('CLI_DESCRIPTION_SEARCHINDEX_LIST'));
```

Reply:
i.e. move to new lines where it increases readability but don't go overboard with it. The `setName` works fine on the same line, so does the semi-colon. Similar comment for the other commands.